client/src/components/pages/application/Profile/index.js

```diff
@@ -1,5 +1,329 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Helmet } from 'react-helmet';
+import Checkbox from '@material-ui/core/Checkbox';
+import { useHistory } from 'react-router-dom';
+import Limitation from '../../../common/limitation';
+import Card from '../../../common/card';
+import Typography from '../../../common/Typography';
+import Button from '../../../common/Button';
+import backGround from '../../../../assets/images/backGround.svg';
+import profileImage from '../../../../assets/images/HomeImg.svg';
+import './index.css';
 
-const Profile = () => <h1>Profile</h1>;
+const applicantData = async (id) => {
+  const {
+    data: { user },
+  } = await axios.get(`/api/v1/applicants/${id}`);
+  return user;
+};
 
-export default Profile;
+const getTec = async () => {
+  const { data } = (await axios.get('/api/v1/tasks')).data;
+  return data;
+};
+
+const updatePoints = async (id) => {
+  const Points = await axios.get(`/api/v1/applicants/${id}/points`);
+  return Points;
+};
+
+const getUserID = async () => {
+  const { data } = await axios.get('/api/v1/isUser');
+  return data;
+};
+
+const ProfilePage = () => {
+  const [data, setData] = useState();
+  const [Technical, setTechnical] = useState();
+  const [UserId, setId] = useState();
+  const history = useHistory();
+
+  useEffect(() => {
+    if (!UserId)
+      getUserID().then((response) => {
+        if (response.message === 'you are authorized') {
+          setId(response.userId);
+        }
+      });
+    if (!data && UserId) {
+      updatePoints(UserId).then();
+      applicantData(UserId).then(setData);
+    }
+    if (!Technical) getTec().then(setTechnical);
+  }, [data, Technical, UserId]);
+
+  return (
+    <div className="Container_page Container_page_profile">
+      <Helmet>
+        <title>Profile</title>
+      </Helmet>
+      <img src={backGround} alt="backGround" className="profile backGround" />
+      <img
+        src={profileImage}
+        alt="backGround"
+        className="profileImage profile"
+      />
+
+      {data && Technical ? (
+        <div className="Profile__container">
+          <Card
+            ClassName="card_profile"
+            content={
+              <table className="Profile__table">
```

Comment:
```suggestion
              <table className="Profile__table">
              <tbody> content </tbody>
              </table>
```
There is a 20 validateDOMNesting Error in the console 
just wrap the content inside the TABLE tag with TBODY tag